app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
In simple words: Administration link is not visible anymore for users with roles of :cfp, :info_desk, :volunteer_coordinator. Before changing `signed_in_with_roles(user)` in ability.rb (line:87), these users were able to see the link. What we want to achieve here? make the link visible again for users with one of above mentioned roles.